CHANGELOG.md

```diff
@@ -0,0 +1,132 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+This file follows the formats and conventions from [keepachangelog.com]
+
+Find Sardana issues in the following locations:
+[old SF sardana issues](https://sourceforge.net/p/sardana/tickets/)
```

Comment:
I would remove the mention to the old SF. In the case of Sardana the new issues already contain links to the SF tickets